src/server.ts

```diff
@@ -51,16 +43,19 @@ const resolvers = {
   },
 
   Mutation: {
-    login: (_, { login }) => {
+    login: async (_, { login }) => {
       const { email, password } = login;
+      const usersRepository = getRepository(User);
 
       if (!email || !password) {
         throw new Error('Unauthorized. Possible invalid credentials.');
       }
 
-      const user = fakeUsers.filter((user) => user.email === email && user.password === password)[0];
+      const user = await usersRepository.findOne({ where: { email } });
```

Comment:
Quando só tem "where", dá pra passar um objeto mais simples, só com os atributos:

```suggestion
      const user = await usersRepository.findOne({ email });
```